### DISCUSSION TITLE
Flight modes missing

### DISCUSSION THREAD

**Discussion by EsotecDevel** [iNavFlight/inav-configurator#2136] [General]
Hello all,
Setting up INAV Configurator 7.1.1 for the first time the presets hung. Tried rebooting, tried both 3 inch and 5 inch MC, same result. Decided to proceed anyway. The presets seem to be in order except for the Modes tab - the Flight Modes are missing. So I guess this is where the preset process got hung up.
Can this be fixed? Or can somebody direct me to the commands needed to set up modes in the CLI?
Thanks in advance,
Phil

**Comment by MrD-RC** [comment]
Flight modes are not set in the presets. If you are missing modes, it is likely because your current setup cannot use them. For example, you will not see navigation modes if you don't have the appropriate hardware.

**Reply by EsotecDevel** [reply]
The relevant navigation modes are listed, but no Flight Modes at all. The list goes:
Arming
Navigation Modes
Flight Mode Modifiers (1 item)
OSD Modes
FPV Camera Modes
Misc Modes

No Flight Modes, sorry.

**Comment by sensei-hacker** [comment]
I have a thought about what could be happening. Can you please show a screenshot your Mixer tab, and your Modes tab. Thanks.

**Comment by EsotecDevel** [comment]
I appreciate the prompt attention. Screenshots attached. As you can see, it is a coaxial with 4 flaps. 
Being a newby with Win 11 there has to be a better way to take screen shots... the mode page continues on the same way.

![Mixer1](https://github.com/iNavFlight/inav-configurator/assets/154710119/6204e8af-eff6-46f5-a036-89753860d0ff)
![Modes1](https://github.com/iNavFlight/inav-configurator/assets/154710119/024491ed-43be-421f-a5c1-3b9fe0874e6a)
![Modes2](https://github.com/iNavFlight/inav-configurator/assets/154710119/e3e5e2ee-1e03-49cd-bdca-8592d8e0e658)

**Comment by sensei-hacker** [comment]
> Being a newby with Win 11 there has to be a better way to take screen shots

The way I normally do it is is the search, type "snip". This show "Snipping Tool" in the menu.

**Comment by sensei-hacker** [comment]
I see the status bar shows "invalid setting".
In your shoes,  if I hadn't done much setup yet I would probably click "reset settings" on the Setup page. Then load a quadcopter preset and modify from there.

Alternatively, if you run "status" and "diff", it may be possible to figure out which setting is invalid.

**Reply by EsotecDevel** [reply]
As I mentioned previously, "reset settings" hangs, does not complete, even after 30 minutes. This occurred for both 3 inch and 4 inch quad.
Tried several times so it probably won't work now and I would prefer not to have to start again. 
I am new to INAV and have never used the CLI. Are you suggesting I just run "status" and "diff" there? Thanks.

**Comment by sensei-hacker** [comment]
Yeah Configurator 7.1.1 has a bug where sometimes it won't close out after it finishes. You can close it after a minute. Or use 7.1.0

Yes, run "status" and "diff" in the CLI.  You can post the output here.

**Comment by EsotecDevel** [comment]
Here we go.

[INAV_7.1.2_cli_20240705_180325.txt](https://github.com/user-attachments/files/16114563/INAV_7.1.2_cli_20240705_180325.txt)

**Comment by sensei-hacker** [comment]
Excellent. In the CLI, you can now do:

set platform_type = MULTOROTOR
save

That will take care of one problem. Hopefully there aren't any other issues.

**Comment by EsotecDevel** [comment]
Lo and behold, all the modes now appear to be present. Cheers for your help, I am pleased to serve as beta tester (for unusual configurations at least).
